src/development/tools/sdk/release-notes/supported-platforms.md

```diff
@@ -4,107 +4,75 @@ short-title: Supported platforms
 description: The platforms that Flutter supports by platform version.
 ---
 
-## Support Tiers
+## Support tiers
 
 We define three tiers of support for the platforms on
-which built Flutter may be deployed:
+which built Flutter might be deployed:
 
-1. Google-tested platforms
-  are automatically tested at every commit
-  by continuous integration testing.
-1. Best-effort platforms, supported through community
+1. **Supported**, Google-tested platforms
+   are automatically tested at every commit
+   by continuous integration testing.
+1. **Best-effort** platforms, supported through community
    testing, are platforms we believe we support through
    coding practices and ad-hoc testing,
    but rely on the community for testing.
```

Comment:
This last clause feels a little odd -- we believe we support it through... testing, but we rely on the community for testing. I understand the point, but perhaps it's better to just say what we do offer, e.g. 

"platforms we intend to work through coding practices, but that we only test on an ad-hoc basis"